include/SSVOpenHexagon/Core/HexagonGame.hpp

```diff
@@ -387,6 +387,7 @@ class HexagonGame
     void drawTrailParticles();
     void drawSwapParticles();
     void drawImguiLuaConsole();
+    void drawMainLayer(const auto getRenderStates);
```

Comment:
Unused, right?

Reply:
True I tried implementing this in another way initially but ended up not using it and forgot to remove it from the header.